go/vt/vttablet/tabletserver/stream_consolidator_test.go

```diff
@@ -97,6 +99,15 @@ func (ct *consolidationTest) leader(stream StreamCallback) error {
 	return nil
 }
 
+func (ct *consolidationTest) waitForResults(worker int, count int64) {
+	for {
+		if atomic.LoadInt64(&ct.results[worker].count) > count {
```

Comment:
Just making sure you intended for `>` and not `>=`?

Reply:
Ayup, it needs to wait until we have received more than the expected amount of results.